docs/receptor.rst

```diff
@@ -0,0 +1,51 @@
+.. _receptor_plugin:
```

Comment:
I don't see how this would be reachable from the docs if it isn't at least added to `index.rst`

Should probably also add a note here that this would require installing Runner on the remote Receptor node also.

Reply:
Not including this in `index.rst` was an oversight.  Will fix.

The doc does say you need Runner on the remote Receptor node, in the section titled `Server Configuration`.